install/include/step3.php

```diff
@@ -50,22 +50,25 @@
 	}
 
 	if (!isset($errors)) {
-		$db = @mysql_connect($_POST['step2']['db_host'] . ':' . $_POST['step2']['db_port'], $_POST['step2']['db_login'], urldecode($_POST['step2']['db_password']));
-		@mysql_select_db($_POST['step2']['db_name'], $db);
+		$db = mysqli_connect($_POST['step2']['db_host'], $_POST['step2']['db_login'], urldecode($_POST['step2']['db_password']) ,$_POST['step2']['db_name']);
+		mysqli_select_db($db, $_POST['step2']['db_name']);
 
 		$status = 3; // for instructor account
 
 		$sql = "INSERT INTO ".$_POST['step2']['tb_prefix']."users (login, password, user_group_id, email, web_service_id, create_date)
 		VALUES ('$_POST[admin_username]', '$_POST[form_admin_password_hidden]', 1, '$_POST[admin_email]', '".sha1(mt_rand() . microtime(TRUE))."', NOW())";
-		$result= mysql_query($sql, $db);
+		$result= mysqli_query($db, $sql);
+ 
+		// $_POST['site_name'] = $addslashes($_POST['site_name']);
+		$_POST['site_name'] =  mysqli_real_escape_string($db, $_POST['site_name']);
```

Comment:
This change is unnecessary with the upgrade of `$addslashes` implementation. This comment applies to other locations too.